src/include/distributed/multi_physical_planner.h

```diff
@@ -321,6 +321,12 @@ typedef struct Task
 	 */
 	char *fetchedExplainAnalyzePlan;
 	int fetchedExplainAnalyzePlacementIndex;
+
+	/*
+	 * Execution Duration fetched from worker. This is saved to be used later by
+	 * ExplainTaskList().
+	 */
+	double fetchedExplainAnalyzeExecutionDuration;
```

Comment:
also update `CopyNodeTask` to include this field.

Reply:
Also to `OutTask`, we seem  to miss some other fields, we can  add them in this PR as well